JetpackStatsWidgetsCore/Package.swift

```diff
@@ -0,0 +1,19 @@
+// swift-tools-version: 5.9
+
+import PackageDescription
+
+let librarySourceName = "JetpackStatsWidgetsCore"
+
+let package = Package(
+    name: librarySourceName,
+    products: [
+        .library(name: librarySourceName, targets: [librarySourceName]),
+    ],
+    targets: [
+        .target(name: librarySourceName),
+        .testTarget(
+            name: "\(librarySourceName)Tests",
+            dependencies: [.target(name: librarySourceName)]
```

Comment:
The template for the packages uses

```swift
            dependencies: ["MyLibrary"]
```

But if you replace `"MyLibrary"` with the name of the library, the code no longer compiles because the expected type is `[Target.Depency]`, not `[String]`.